website/src/docs/aws-s3-multipart.md

```diff
@@ -109,13 +109,12 @@ A function that generates a batch of signed URLs for the specified part numbers.
 
 * `uploadId` - The UploadID of this Multipart upload.
 * `key` - The object key in the S3 bucket.
-* `partNumbers` - An array of indices of this part in the file (`PartNumber` in S3 terminology). Note that part numbers are _not_ zero-based.
-* `chunks` - A Javascript object with the part numbers as keys and the Blob data for each part as the value.
+* `parts` - An array of objects with the part number and chunk (`Array<{ number: number, chunk: blob }>`). `number` can’t be zero.
```

Comment:
has there been consideration for using `partNumber` in place of `number`? So it would read as `Array<{ partNumber: number, chunk: blob }>` instead.

It might just be me, but reading `{ number: number }` kind of trips one up a little bit at first glance.

Reply:
Personally, I prefer making keys intention revealing with the context of the parent object. `part.number` already implies it's the part's number, wheres `part.partNumber` sounds like a pleonasm to me.